### PR TITLE
Feature v0.12.0 node acls

### PR DESCRIPTION
**Pull request by 0xdcarns**

### CODE REVIEW

File: controllers/config/dnsconfig/netmaker.hosts [hunk 1]
```diff
@@ -1,2 +1 @@
-10.0.0.1         testnode.skynet
-10.0.0.2         myhost.skynet
+10.0.0.2         testnode.skynet myhost.skynet
```
Comment (mattkasun) [inline controllers/config/dnsconfig/netmaker.hosts:1]: add controllers/config/dnsconfig/* to .gitignore

File: logic/nodes.go [hunk 2]
```diff
@@ -274,6 +311,13 @@ func CreateNode(node *models.Node) error {
 	if err != nil {
 		return err
 	}
+	// TODO get template logic to decide initial ACL value
```
Comment (mattkasun) [inline logic/nodes.go:314]: Is this comment still valid?

File: netclient/functions/mqhandlers.go [hunk 3]
```diff
@@ -196,16 +196,17 @@ func UpdatePeers(client mqtt.Client, msg mqtt.Message) {
...
 	if cfg.Node.NetworkSettings.IsComms == "yes" {
 		return
 	}
-	ncutils.Log("DNS updating /etc/hosts")
 	if cfg.Node.DNSOn == "yes" {
 		if err := setHostDNS(peerUpdate.DNS, ncutils.IsWindows()); err != nil {
 			ncutils.Log("error updating /etc/hosts " + err.Error())
 			return
 		}
+		ncutils.Log("DNS updating /etc/hosts")
```
Comment (mattkasun) [inline netclient/functions/mqhandlers.go:209]: This log message is probably not required given the peer update log message earlier

**Review by mattkasun — CHANGES_REQUESTED** [review]

File: logic/nodes.go (hunk 2 above)
Comment (0xdcarns) [reply logic/nodes.go:314]: yes, for future network templates

**Review by mattkasun — APPROVED** [review]

**Pull request merged**